### ISSUE TITLE
Python: missing class information with "class" on multiple lines

### ISSUE THREAD

**Issue by blueyed** [universal-ctags/ctags#1103]
The "class" information is missing with class `A` below, because its definition
is across multiple lines:

```sh
% cat t.py
class A(Foo,
        Bar):
    a = 1

class B():
    b = 2

% ./ctags -f- t.py
A	t.py	/^class A(Foo,$/;"	c	language:Python
B	t.py	/^class B():$/;"	c	language:Python
a	t.py	/^    a = 1$/;"	v	language:Python
b	t.py	/^    b = 2$/;"	v	language:Python	class:B
```

**Comment by b4n** [comment]
Odd and interesting, I'll take a look.

**Comment by b4n** [comment]
OK it was a fairly simple mistake in the line continuation support.  Fixed in #1105.
